src/js/hca-rjsf/config/form.js

```diff
@@ -670,31 +672,50 @@ const formConfig = {
             'view:spouseIncome': {
               'ui:title': 'Spouse income',
               'ui:options': {
-                hideIf: (formData) => formData.maritalStatus !== 'married' // Something else too?
+                hideIf: (formData) => !formData.maritalStatus || (formData.maritalStatus.toLowerCase() !== 'married') // Something else too?
```

Comment:
Yeah, that's a good point. I forgot we were looking for this in another place as well. I'll fix that up while I'm here.